app/controllers/concerns/accessible.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Accessible
```

Comment:
I do not think we need this module at all, also it rather should go to the app/concerns folder, not controllers :)